src/python/pants/backend/codegen/tasks/simple_codegen_task.py

```diff
@@ -6,6 +6,7 @@
                         unicode_literals, with_statement)
 
 import os
+from abc import ABCMeta, abstractmethod
```

Comment:
Pants has a wrapper around ABCMeta that you might be able to substutute: AbstractClass in `src/python/pants/util/meta.py`